### ISSUE TITLE
[Channel] Delete message parameter is strange

### ISSUE THREAD

**Issue by DonWada** [sendbird/sendbird-ios-framework#51]
To delete a message in a channel, the SDK asked for a SBDBaseMessage object, which is counter-intuitive, because:
- There are many different ways to pass data from the model layer (SBDBaseMessage) to the UI layer, and the SBDBaseMessage is probably "lost" at some point.
- There is no way to fetch a SBDBaseMessage object by id directly.

The most natural way would be to ask for the message id (Int64) as a parameter.

Proof is that to delete a message, this is the code I use (and which works):

```swift
let message = SBDBaseMessage()
message.messageId = id
channel.delete(message)
```

I end up creating a "fake" unmanaged SBDBaseMessage object, which I set its id. And it works normally.

This confirms that inside your SDK, you are probably doing:

```swift
let messageId = baseMessage.id
```

Therefore the object is never used nor needed, only the id.

**Comment by intoxicated** [comment]
@DonWada FYI, we will include this feature on next release (ETA end of April)

**Comment by intoxicated** [comment]
@DonWada It's been released in 3.0.179 FYI.
